docs/intro/reusable-apps.txt

```diff
@@ -314,6 +314,72 @@ working. We'll now fix this by installing our new ``django-polls`` package.
 
 #. Run the development server to confirm the project continues to work.
 
+Testing your app
+================
+
+Since we moved the ``polls`` directory out of the project, we don't have the
+project's ``settings.py`` and ``urls.py`` anymore. We need another way to
+configure our app in order to test it.
+
+#. First create a ``tests`` directory and move the tests over from the
+   ``polls`` directory. Because the tests now reside in a different module the
+   relative import ``from .models import Poll`` has to be changed to
+   ``from polls.models import Poll``.
+
+#. Now create a ``settings.py`` inside the ``tests`` folder which will contain
+   only the settings required to test the application.
+
+   .. code-block:: python
+       :caption: ``tests/settings.py``
+
+       from django.urls import include, path
+
+       SECRET_KEY = "fake-key"
+
+       INSTALLED_APPS = [
+           "polls",
+           # adding tests allows us to define models inside the tests package
+           # "tests",
```

Comment:
```suggestion
           "tests",  # Optional: adding tests allows us to define models inside the tests directory.
```
(maybe) note that this file is also in the advanced topic so if we chose to have this in both places, this comment applies there too